docs/book/pattern/capture-cache.md

```diff
@@ -42,9 +46,36 @@ Option | Data Type | Default Value | Description
 `public_dir` | `string` | none | Location of the public web root directory in which to write output.
 `index_filename` | `string` | "index.html" | The name of the index file if only a directory was requested.
 `file_locking` | `bool` | `true` | Whether or not to lock output files when writing.
-`file_permission` | `int | bool` | `0600` (`false` on Windows) | Default permissions for generated output files.
-`dir_permission` | `int | bool` | `0700` (`false` on Windows) | Default permissions for generated output directories.
-`umask` | `int` | `bool` | `false` | Whether or not to umask generated output files / directories.
+`file_permission` | `int\|false` | `0600` (`false` on Windows) | Default permissions for generated output files.
+`dir_permission` | `int\|false` | `0700` (`false` on Windows) | Default permissions for generated output directories.
+`umask` | `int\|false` | `false` | Whether or not to umask generated output files / directories.
+
+## Examples
+
+### Scaling Images in the Web Root
+
+Using the following Apache 404 configuration:
+
+```apacheconf
+# .htdocs
+ErrorDocument 404 /index.php
+```
+
+Use the following script:
+
+```php
+// index.php
+use Laminas\Cache\Pattern\CaptureCache;
+use Laminas\Cache\Pattern\PatternOptions;
+
+$capture = new CaptureCache(
+    new PatternOptions([
+        'public_dir' => __DIR__,
+    ])
+);
+
+// TODO
```

Comment:
Instead of adding a todo comment in the documentation, a new issue report should be created. Or is this an old comment?

Reply:
Uhm, good question TBH. Have to double check 😅